source/contentRecog/recogUi.py

```diff
@@ -74,7 +74,7 @@ def setFocus(self):
 			# we want the cursor to move to the focus (#3145).
 			# However, we don't want this for recognition results, as these aren't focusable.
 			ti._enteringFromOutside = True
-		eventHandler.executeEvent("gainFocus", self)
+		eventHandler.queueEvent("gainFocus", self)

```

Comment:
Might be worth having a comment here to say why we do this. You can probably pretty much copy your commit message.